README.md

```diff
@@ -66,9 +67,24 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
 
 After installing the RPM, you can use the modules in the same way as when they are installed directly from Ansible Galaxy.
 
+## Using the modules
+
+Please see [the Plugin Index](https://theforeman.org/plugins/foreman-ansible-modules/) for the list of plugins in the theforeman.foreman collection.
```

Comment:
how is that different to what we have in https://github.com/theforeman/foreman-ansible-modules/blob/387aa0c23f3ab523b25a24e87047a55ea5566463/README.md#documentation? quoting:

```
A list of all modules and their documentation can be found at theforeman.org/plugins/foreman-ansible-modules.
```

Reply:
I know it's a repetition, I wanted to drive the point home since "how to use this" does came up from time to time when I pointed people to the main page.